### PR TITLE
cukinia: add new recipe

### PR DESCRIPTION
**Pull request by eloi29**
This recipe adds cukinia, a simple validation framework for Linux
systems, to Yocto.

### CODE REVIEW

File: conf/layer.conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "validation"
+BBFILE_PATTERN_validation = "^${LAYERDIR}/"
+BBFILE_PRIORITY_validation = "6"
```
Comment (d-k-c) [inline conf/layer.conf:10]: Please add LAYERSERIES_COMPAT_validation to indicate the versions you're compatible with: https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-LAYERSERIES_COMPAT

File: recipes-support/cukinia/cukinia_git.bb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+SUMMARY = "cukinia"
+DESCRIPTION = "Cukinia is designed to help Linux-based embedded firmware developers run simple validation tests on their firmware"
+HOMEPAGE = "https://github.com/savoirfairelinux/cukinia"
+LICENSE = "Apache-2.0|GPL-3.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
+
+SRCREV = "84189fe565d421683156bff9e2d466b794b63373"
```
Comment (d-k-c) [inline recipes-support/cukinia/cukinia_git.bb]: Move that below SRC_URI

File: recipes-support/cukinia/cukinia_git.bb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+SUMMARY = "cukinia"
+DESCRIPTION = "Cukinia is designed to help Linux-based embedded firmware developers run simple validation tests on their firmware"
+HOMEPAGE = "https://github.com/savoirfairelinux/cukinia"
+LICENSE = "Apache-2.0|GPL-3.0"
```
Comment (d-k-c) [inline recipes-support/cukinia/cukinia_git.bb]: Add whitespaces around the "|"

**Review by d-k-c — COMMENTED** [review]
A few nitpicky comments. Also, if you're compatible with some versions of oecore, make sure to create the corresponding branches.

File: conf/layer.conf [hunk 4]
```diff
@@ -8,3 +8,4 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "validation"
 BBFILE_PATTERN_validation = "^${LAYERDIR}/"
 BBFILE_PRIORITY_validation = "6"
+LAYERSERIES_COMPAT_yoctobsp = "rocko"
```
Comment (d-k-c) [inline conf/layer.conf]: yoctobsp? It should match your layer name, not some other layer's name

File: recipes-support/cukinia/cukinia_git.bb [hunk 5]
```diff
@@ -1,11 +1,12 @@
 SUMMARY = "cukinia"
 DESCRIPTION = "Cukinia is designed to help Linux-based embedded firmware developers run simple validation tests on their firmware"
 HOMEPAGE = "https://github.com/savoirfairelinux/cukinia"
-LICENSE = "Apache-2.0|GPL-3.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
```
Comment (d-k-c) [inline recipes-support/cukinia/cukinia_git.bb]: These two lines were at the right place

File: conf/layer.conf (hunk 4 above)
Comment (eloi29) [reply conf/layer.conf]: arf sorry. I made a copy paste error

**Review by d-k-c — APPROVED** [review]

**Pull request merged**